temporalio/bridge/src/client.rs

```diff
@@ -50,12 +51,21 @@ struct ClientRetryConfig {
     pub max_retries: usize,
 }
 
+#[derive(FromPyObject)]
+struct RpcCall {
+    rpc: String,
+    req: Vec<u8>,
+    retry: bool,
+    metadata: HashMap<String, String>,
+    timeout_millis: Option<u64>,
```

Comment:
This could just be a duration directly, if pyo3 does that.

Reply:
It doesn't. Ref https://pyo3.rs/v0.16.4/conversions/tables.html. I could take a timedelta directly and convert myself, but this is close enough and ms is decent resolution (note I have done this with all other durations to Rust).